document about GitHub Action (#50)
deep-diver/semantic-segmentation-ml-pipeline: .github/workflows/README.md

There are two GitHub Action pipelines for [Continuous Integration(CI)](ci.yml) and [Continuous Deployment(CD)](cd-training-pipeline.yml) of the ML pipeline on Vertex AI environment. 

> **CI**: This is to verify the proposed changes of the pipeline by creating and compiling through [TFX CLI](https://www.tensorflow.org/tfx/guide/cli). Specifically, `tfx pipeline create` and `tfx pipeline compile` returns errors if the codes of the pipeline has import/syntax problems. 

Reviewer: ```suggestion
**CI**: This is to verify the proposed changes of the pipeline by creating and compiling through the [TFX CLI](https://www.tensorflow.org/tfx/guide/cli). Specifically, `tfx pipeline create` and `tfx pipeline compile` will error out if the given pipeline code has import/syntax problems. 
```

> - You can furthure improve CI pipeline by actually run the pipeline locally with `tfx run create`. However, in this case, you need to consider the code structure to be able to set epochs and datasets differently. You certainly don't want to run the pipeline over the long epochs and full dataset since the purpose of CI is to check if there is no problem.

Reviewer: ```suggestion
You can also _run_ the pipeline locally with `tfx run create`. However, in this case, you need to consider the code structure to be able to set the number of epochs and datasets differently. You certainly don't want to run the pipeline over for a larger number of epochs and on the full dataset since the purpose of CI is to check if there is no problem.
```

- `gcpRegion`: This sets which GCP Region to run the pipeline on Vertex AI. The default value is set to `un-central1`.
- `pipelineName`: In order to run the pipeline, TFX CLI need to know which pipeline to run. The names of each pipeline are defined in `configs.py` files, and these values are rigistered automatically when running `tfx pipeline create` CLI. In this project, there is only one pipeline, and its name is set to `segmentation-training-pipeline` by default.
- `enableDataflow`: This option is used to delegate the jobs of `ExampleGen` and `Transform` to [DataFlow](https://cloud.google.com/dataflow?hl=ko) service. This is useful to handle a large amount of data since the VM spec of each step in Vertex AI Pipeline is limited, and you sometimes get Out-Of-Memory(OOM) issue.

> The basic structure of CD pipeline runs `tfx pipeline create`, `tfx pipeline compile`, and `tfx run create` CLIs sequentially. Here are the purpose of each CLIs:

Reviewer: ```suggestion
The basic structure of CD pipeline runs `tfx pipeline create`, `tfx pipeline compile`, and `tfx run create` CLIs sequentially:
```
